fastify.d.ts

```diff
@@ -7,7 +7,6 @@
 import * as http from 'http'
 import * as http2 from 'http2'
 import * as https from 'https'
-import * as tls from 'tls'
```

Comment:
why was this removed?

Reply:
I think it was not used in the file and my editor removed it automatically. Is there a reason to keep it?